Python/Product/PythonTools/PythonTools/Commands/ExecuteInReplCommand.cs

```diff
@@ -211,7 +211,7 @@ internal sealed class ExecuteInReplCommand : Command {
             var eval = (IPythonInteractiveEvaluator)window.InteractiveWindow.Evaluator;
 
             // The interpreter may take some time to startup, do this off the UI thread.
-            await ThreadHelper.JoinableTaskFactory.RunAsync(async () => {
+            await _serviceProvider.GetPythonToolsService().UIThread.InvokeAsync(async () => {
```

Comment:
what's the benefit here?

Reply:
ThreadHelper library isn't available during unit tests. It was causing exceptions when running.

Reply:
PythonToolsService.UIThread is using a MockUIThread during testing.